src/pages/slides/steffen/intro.js

```diff
@@ -0,0 +1,259 @@
+          <ul>
+            <li className='fragment'>
+              Installiert Git und checkt die Repos aus
+            </li>
+            <li className='fragment'>
+              Installiert JDK und JRE
+            </li>
+            <li className='fragment'>
+              Installiert und Konfiguriert eure IDE
+            </li>
+            <li className='fragment'>
+              Entwickelt alles lokal an eurem Rechner
+            </li>
+          </ul>
+        </section>
+      </section>
+      <section>
+        <section>
+          <h2>Was sind Programme?</h2>
+        </section>
+        <section>
+          <h2>Verschiedene Arten</h2>
+          <li className='fragment'>
+            Programme mit GUI
+          </li>
+          <li className='fragment'>
+            Hintergrundprogramme
+          </li>
+          <li className='fragment'>
+            Programme mit TUI
+          </li>
+        </section>
+        <section>
+          <h2>Kommandozeile</h2>
+          <p className='fragment'>
+            Syntax: &lt;name&gt; [OPTION, ...] [--flag, ...]
+          </p>
+          <pre className='fragment'>
+            <code className='bash'  dangerouslySetInnerHTML={{ __html: 
+              'ls # alle Ordner und Dateien anzeigen \n' +
+              'ls -l # wie Z1, aber als Liste anzeigen \n' + 
+              'ls -la # wie Z2, aber auch versteckte dateien und ordner \n' +
+              'ls docs -la # wie Z3, aber im Unterordner docs \n' 
+              }} ></code>
+          </pre>
+        </section>
+        <section>
+          <h2>Wie macht man ein Java Programm?</h2>
+          <p className='fragment'>
+            Quellcode verfassen
+          </p>
+          <p className='fragment'>
+            Quellcode zu einem Java Programm compilieren
+          </p>
+          <p className='fragment'>
+            Java Programm mit der Java Runtime ausführen
+          </p>
+          <aside class="notes">
+            <h3>Beispiel: Hello World</h3>
+            <ul>
+              <li>javac</li>
+              <li>java Exercise.java</li>
+              <li>String[] args in debugger</li>
+            </ul>
+          </aside>
+        </section>
+      </section>
+      <section>
+        <section>
+          <h2>Zusammenfassung</h2>
+        </section>
+        <section>
+          <h2>Programme</h2>
+          <p className='fragment'>
+            GUI, TUI &amp; Hintergrund
+          </p>
+          <p className='fragment'>
+            Quellcode wird in Programm compiliert
+          </p>
+        </section>
+        <section>
+          <h2>Aufgaben</h2>
+          <p className='fragment'>
+            Development Environment einrichten (Gitpod oder Lokal)
+          </p>
+          <p className='fragment'>
+            <a 
+              target='_blank'
+              href='https://appenmaier.github.io/programming/exercises/class-structure/class-structure01'
+            >
+              Hello World aufgabe machen
```

Comment:
```suggestion
              Hello-World-Aufgabe machen
```
Oder falls dir das nicht so gut gefällt: Hello World Aufgabe machen

Hauptsache Aufgabe ist groß geschrieben